assets/airports/kcvg.json

```diff
@@ -0,0 +1,256 @@
+{
+	"reference": "https://nfdc.faa.gov/nfdcApps/services/airportLookup/airportDisplay.jsp?airportId=KCVG",
+	"reference2": "https://skyvector.com/airport/CVG/Cincinnati-Northern-Kentucky-International-Airport",
+	"radio": {
+		"twr": "Cincinnati Tower",
+		"app": "Cincinnati Approach",
+		"dep": "Cincinnati Departure"
+	},
+	"icao": "KCVG",
+	"iata": "CVG",
+	"magnetic_north": -4,
+	"ctr_radius": 80,
+	"ctr_ceiling": 20000,
+	"initial_alt": 10000,
+	"position": ["N39d2.93", "W084d40.07", "896ft"],
+	"rr_radius_nm": 80,
+	"rr_center": ["N39d00.96", "W84d42.20"],
+	"has_terrain": false,
+	"wind": {
+		"angle": 0,
+		"speed": 0
+	},
+	"fixes": {
```

Comment:
I would highly suggest alphabetizing these fix names.  That will make it much easier to find a fix in the list.